README.md

```diff
@@ -38,7 +38,15 @@
 
 ## ⚙️ Installation
 
-Make sure you have Go installed ([download](https://go.dev/dl/)). Version `1.18` or higher is required.
+### Dependencies
+
+- Golang 1.18 or higher is required to build and run the project.You can found the installer on the official Golang
```

Comment:
```suggestion
- Golang 1.18 or higher is required to build and run the project. You can find the installer on the official Golang
```

Reply:
This page should be linked https://go.dev/doc/install

Reply:
Done